cypress/support/index.d.ts

```diff
@@ -3,5 +3,13 @@ declare namespace Cypress {
 
   interface Chainable<Subject> {
     mount(children: React.ReactNode): Cypress.Chainable<MountReturn>;
+    realMount(children: React.ReactNode): Cypress.Chainable<MountReturn>;
```

Comment:
Interestingly enough I don't believe you're returning a MountReturn in `realMount` (which is fine, I doubt we'll super need it, and we can come back and tweak if if we do in the future), so for clarity you may want to remove the return typing

```suggestion
    realMount(children: React.ReactNode): void;
```